pandas/io/sql.py

```diff
@@ -2411,7 +2413,15 @@ def to_sql(
 
     def has_table(self, name: str, schema: str | None = None) -> bool:
         wld = "?"
-        query = f"SELECT name FROM sqlite_master WHERE type='table' AND name={wld};"
+        query = f"""
```

Comment:
is something like this equivalent here opposed to the sql? 
```
self.meta.reflect(bind=self.con)
return name in metadata.tables.keys()
```

Reply:
Yes, I believe so https://www.sqlite.org/schematab.html

Since this is the sqlite3 backend unfortunately we can't `meta.reflect`